addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -140,7 +150,6 @@
 <template name="Display 'Buy Now'" id="course_option_buy_course_now" inherit_id="website_sale_slides.course_buy_course_button" active="False">
     <xpath expr="//div[hasclass('add_to_cart_button')]" position="before">
         <div class="mb-1">
-          <!-- FIXME TDE seems strange to redirect to /cart/update with get params when it only supports POST requests -->
```

Comment:
Could switch that comment to "The 'post_link' class will automagically transform the GET request into a POST".

Or something along those lines, to avoid any further confusion on this.

Reply:
Not sure it's worth it honestly, you can just keep it as is then.